avroproducer/src/main/java/com/epam/learning/backendservices/AvroProducer.java

```diff
@@ -0,0 +1,44 @@
+package com.epam.learning.backendservices;
+
+import org.apache.kafka.clients.producer.*;
+import com.epam.learning.backendservices.kafka.BuiRegistration;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import java.util.logging.Logger;
+
+public class AvroProducer {
+
+    private static final Logger logger = Logger.getLogger(AvroProducer.class.getName());
+
+    public static void main(String[] args) throws IOException {
+        InputStream inputStream = AvroProducer.class.getClassLoader().getResourceAsStream("avroproducer.properties");
+        Properties properties = new Properties();
+        properties.load(inputStream);
+
+        KafkaProducer<String, BuiRegistration> kafkaProducer = new KafkaProducer<>(properties);
+        String topic = "bui_kafka_topic";
```

Comment:
Optional: include topic name in properties file too.

Reply:
Done